pkg/controller/remotewatcher/remotewatcher_controller.go

```diff
@@ -172,6 +173,17 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// Job
+	err = c.Watch(
+		&source.Kind{
+			Type: &batchv1.Job{},
+		},
+		&handler.EnqueueRequestForObject{})
```

Comment:
We need a predicate here, which will separate our playbook hook jobs from all the others.

Reply:
Agree, just haven't gotten there yet.